Description: quick fix for some problematic profiles
 The stable version (this) SuperSlicer does not handle some new profile
 settings despite release notes to the contrary. We patch the troublesome
 profiles to keep the program from generating a segmentation fault.
Author: <Aegean Odyssey> 57586704+aegean-odyssey@users.noreply.github.com
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: superslicer/resources/profiles/Anycubic.ini
===================================================================
--- superslicer.orig/resources/profiles/Anycubic.ini
+++ superslicer/resources/profiles/Anycubic.ini
@@ -1980,7 +1980,7 @@ compatible_printers_condition = printer_
 complete_objects = 0
 default_acceleration = 900
 dont_support_bridges = 1
-draft_shield = disabled
+draft_shield = 0
 elefant_foot_compensation = 0.2
 ensure_vertical_shell_thickness = 1
 external_perimeter_extrusion_width = 0
Index: superslicer/resources/profiles/PrusaResearch.ini
===================================================================
--- superslicer.orig/resources/profiles/PrusaResearch.ini
+++ superslicer/resources/profiles/PrusaResearch.ini
@@ -8898,7 +8898,7 @@ thumbnails = 16x16,220x124
 bed_shape = 0x0,180x0,180x180,0x180
 default_filament_profile = "Prusament PLA"
 default_print_profile = 0.15mm QUALITY @MINI
-gcode_flavor = marlin2
+gcode_flavor = marlin
 machine_max_acceleration_e = 5000
 machine_max_acceleration_extruding = 1250
 machine_max_acceleration_retracting = 1250
Index: superslicer/resources/profiles/Snapmaker.ini
===================================================================
--- superslicer.orig/resources/profiles/Snapmaker.ini
+++ superslicer/resources/profiles/Snapmaker.ini
@@ -44,7 +44,7 @@ clip_multipart_objects = 1
 complete_objects = 0
 default_acceleration = 0
 dont_support_bridges = 1
-draft_shield = disabled
+draft_shield = 0
 first_layer_size_compensation = -0.1
 ensure_vertical_shell_thickness = 1
 external_perimeter_extrusion_width = 0.45
Index: superslicer/resources/profiles/Ultimaker.ini
===================================================================
--- superslicer.orig/resources/profiles/Ultimaker.ini
+++ superslicer/resources/profiles/Ultimaker.ini
@@ -48,7 +48,7 @@ compatible_printers_condition =
 complete_objects = 0
 default_acceleration = 0
 dont_support_bridges = 0
-draft_shield = disabled
+draft_shield = 0
 elefant_foot_compensation = 0
 ensure_vertical_shell_thickness = 0
 external_perimeter_extrusion_width = 0.45
@@ -151,7 +151,7 @@ support_material_extrusion_width = 0.4
 support_material_interface_contact_loops = 0
 support_material_interface_extruder = 1
 support_material_interface_layers = 2
-support_material_interface_pattern = auto
+support_material_interface_pattern = rectilinear
 support_material_interface_spacing = 0.2
 support_material_interface_speed = 100%
 support_material_pattern = rectilinear-grid
Index: superslicer/resources/profiles/Zonestar.ini
===================================================================
--- superslicer.orig/resources/profiles/Zonestar.ini
+++ superslicer/resources/profiles/Zonestar.ini
@@ -77,7 +77,7 @@ clip_multipart_objects = 1
 complete_objects = 0
 default_acceleration = 0
 dont_support_bridges = 1
-draft_shield = disabled
+draft_shield = 0
 elefant_foot_compensation = 0
 ensure_vertical_shell_thickness = 0
 external_perimeter_extrusion_width = 0.45
@@ -392,7 +392,7 @@ default_print_profile =
 deretract_speed = 
 end_gcode = 
 extra_loading_move = -2
-gcode_flavor = marlin2
+gcode_flavor = marlin
 high_current_on_filament_swap = 0
 host_type = octoprint
 layer_gcode = 
